pylint/checkers/classes/class_checker.py

```diff
@@ -258,6 +264,30 @@ def _has_different_parameters(
     return result
 
 
+def _has_different_keyword_parameters(
```

Comment:
To match the docstring?
```suggestion
def _has_different_keyword_only_parameters(
```